src/Stripe.net/Services/Invoices/InvoiceCreateOptions.cs

```diff
@@ -40,6 +39,16 @@ public class InvoiceCreateOptions : BaseOptions
         [JsonProperty("customer")]
         public string CustomerId { get; set; }
 
+        /// <summary>
+        /// Either <c>charge_automatically</c>, or <c>send_invoice</c>. When
+        /// charging automatically, Stripe will attempt to pay this invoice
+        /// using the default source attached to the customer. When sending an
+        /// invoice, Stripe will email this invoice to the customer with payment
+        /// instructions. Defaults to <c>charge_automatically</c>.
+        /// </summary>
+        [JsonProperty("collection_method")]
+        public string CollectionMethod { get; set; }
+
```

Comment:
Sorry I didn't flag them all originally and wanted you to recheck each one. That one should be before `Customer` and `CustomFields`